docs/current_docs/manuals/developer/cookbook/snippets/builds/oci/go/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"context"
+	"time"
+)
+
+type MyModule struct{}
+
+// Build and publish image with oci annotations
+func (m *MyModule) Build(
+	ctx context.Context,
+) (string, error) {
+	ref, err := dag.Container().
+		From("alpine").
+		WithLabel("org.opencontainers.image.title", "my-alpine").
+		WithLabel("org.opencontainers.image.version", "1.0").
+		WithLabel("org.opencontainers.image.created", time.Now().String()).
+		WithLabel("org.opencontainers.image.source", "https://github.com/alpinelinux/docker-alpine").
+		WithLabel("org.opencontainers.image.licenses", "MIT").
```

Comment:
Note: labels are *not* the same as OCI annotations. Even though the same key/value pairs are allowed.

Labels are attached to the [config object](https://github.com/opencontainers/image-spec/blob/main/config.md#properties), while annotations are attached to a [manifest](https://github.com/opencontainers/image-spec/blob/main/manifest.md)(or descriptor). This is fine (and we should aim to support annotations as well), but we should take care to call these labels.

Reply:
Thanks @jedevc - this was a straight port from this entry, so it seems like it was wrong there too? https://archive.docs.dagger.io/0.9/cookbook/#add-oci-annotations-to-image 

Ill update the description in the text to make this part clear.

Reply:
Yup, looks like the old cookbook is wrong too :scream: